test/e2e_node/services/kubelet.go

```diff
@@ -194,17 +194,14 @@ func (e *E2EServices) startKubelet() (*server, error) {
 		cmdArgs = append(cmdArgs,
 			systemdRun,
 			"-p", "Delegate=true",
+			"-p", "StandardError=file:"+framework.TestContext.ReportDir+"/kubelet.log",
```

Comment:
Machines running with enabled SELinux may not have the right policy in place for this change. Everyone who encounters an audit log issue like this:

```
Aug 03 09:45:13 localhost kernel: audit: type=1400 audit(1627983913.613:743): avc:  denied  { write } for  pid=3101 comm="(kubelet)" name="kubelet.log" dev="tmpfs" ino=775 scontext=system_u:system_r:init_t:s0 tcontext=unconfined_u:object_r:user_tmp_t:s0 tclass=file permissive=0
Aug 03 09:45:13 localhost audit[3101]: AVC avc:  denied  { write } for  pid=3101 comm="(kubelet)" name="kubelet.log" dev="tmpfs" ino=775 scontext=system_u:system_r:init_t:s0 tcontext=unconfined_u:object_r:user_tmp_t:s0 tclass=file permissive=0
```

May fix the issue with compiling the SELinux policy `kubelet-e2e.te`:
```te

module kubelet-e2e 1.0;

require {
        type iptables_t;
        type cgroup_t;
        type user_tmp_t;
        type init_t;
        class dir ioctl;
        class file { append create lock map open read rename unlink write };
}

#============= init_t ==============

#!!!! This avc is allowed in the current policy
allow init_t user_tmp_t:file { append create lock map open read rename unlink write };

```

Then compiling and loading the policy:
```bash
# Compile the module
checkmodule -M -m -o kubelet-e2e.mod kubelet-e2e.te

# Create the package
semodule_package -o kubelet-e2e.pp -m kubelet-e2e.mod

# Load the module into the kernel
semodule -i kubelet-e2e.pp
```